web-console/e2e-tests/component/datasources/overview.ts

```diff
@@ -116,6 +124,23 @@ export class DatasourcesOverview {
 
     const editActions = await this.page.$$('span[icon=wrench]');
     editActions[index].click();
+    await this.waitForPopupMenu();
+  }
+
+  private async waitForPopupMenu(): Promise<void> {
+    await this.page.waitFor(1000); // https://blueprintjs.com/docs/#core/components/popover.testing
```

Comment:
Do you think that we should make an allowance in the UI and have a 'testing' mode that zeroes out all the delays in all the popovers in the UI `hoverCloseDelay={0} hoverOpenDelay={0}`. I am imagining that you could evaluate some sort of `enterTestingMode()` function in the app context and it will do this and other things. Not for this PR but wanted to get your thoughts on it.

Reply:
After incorporating the changes with the upgraded playwright version, it seems this fixed sleep is no longer needed to make the E2E test reliable, so I removed it.

Although it may not be needed for E2E tests, I think your suggestion to have a testing mode for components with `Popover`s could be useful for unit tests though.